src/App.tsx

```diff
@@ -1,10 +1,57 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Header } from './Components/Header/Header';
+import { Todo } from './types/Todo';
+import { Footer } from './Components/Footer/Footer';
+import { TodoList } from './Components/TodoList/TodoList';
+import { Errors } from './Components/Errors/Errors';
+
+function idTodo(todos: Todo[]) {
+  const maxId = Math.max(...todos.map(todo => todo.id));
+
+  return maxId + 1;
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState('all');
```

Comment:
Create enum for filter value
```suggestion
  const [status, setStatus] = useState(Filter.All);
```